Gemfile.lock

```diff
@@ -176,6 +177,4 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
```

Comment:
I noticed that this appears in some branches and in some it is not there. Is this linked to some gem maybe?